rfcs/menu-component.md

```diff
@@ -1,8 +1,81 @@
 # Menu
 
-## User Story
-
-> Given an array of items, I want to render them as a list, navigate through it using keyboard bindings, and receive feedback when I select (via pointer select or Enter) an item.
+## Usage
+
+Basic Usage:
```

Comment:
Let's do a subtitle of it, `### Basic Usage`, and another `### Advanced usage` for the override case to align with the rest RFCs.